Sources/SwiftSyntax/SwiftSyntaxCompatibility.swift

```diff
@@ -50,6 +50,32 @@ public extension DeclGroupSyntax {
   }
 }
 
+
+public extension EffectSpecifiersSyntax {
+  @available(*, deprecated, message: "use throwsClause.throwsSpecifier")
+  var throwsSpecifier: TokenSyntax? {
+    throwsClause?.throwsSpecifier
+  }
```

Comment:
This should also provide a setter that sets the `throwsSpecifier` inside the `throwsClause` and creates a `throwsClause` if none exists.

Reply:
That's a good point